src/libmdb/backend.c

```diff
@@ -155,6 +155,37 @@ enum {
 
 static void mdb_drop_backend(gpointer key, gpointer value, gpointer data);
 
+
+gchar *passthrough_unchanged(const gchar *str) {
```

Comment:
This function and the next two should be `static` if they're not used outside of backend.c

Reply:
Done!